scripts/oldump.sh

```diff
@@ -25,9 +25,15 @@
 # 110 minutes to extract the 29GB of data from the database so it is highly
 # recommended to save a copy of data.txt.gz in another directory to accelerate the
 # testing of subsequent job steps.  See `TESTING:` comments below.
+# 
```

Comment:
rm trailing whitespace for pre-commit
```suggestion
#
```